### PR TITLE
add makefile to install tool and port param

### PR DESCRIPTION
**Pull request by jgimeno**

### CODE REVIEW

**Review by faddat — APPROVED** [review]
Makes sense on a ton of levels.

The port flag is great because busy servers tend to get kind of poor, port-wise.

File: README.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ The standalone executable will talk to your blockchain's API and serve as an API
 | Task  | Flag  | Description |
 |---|---|---|
 | Help | --help | explain these flags |
+| Port | --port 8080  | the port where the service is listening |
```
Comment (alessio) [inline README.md:22]:
```suggestion
| Port | -port 8080  | the port where the service is listening |
```

Comment (alessio) [reply README.md:22]: All flags are prefixed wirth one dash '-' character

File: cosmos/launchpad/launchpad.go [hunk 2]
```diff
@@ -61,7 +61,7 @@ func NewLaunchpadNetwork(options Options) service.Network {
 	return service.Network{
 		Properties: rosetta.NetworkProperties{
 			Blockchain:          options.Blockchain,
-			Network:             options.Blockchain,
+			Network:             options.Network,
```
Comment (sahith-narahari) [inline cosmos/launchpad/launchpad.go:64]: nice catch

**Review by sahith-narahari — APPROVED** [review]

**Pull request merged**